README.md

```diff
@@ -7,6 +7,7 @@
 The name `996.ICU` refers to **"Work by '996', sick in ICU"**, an ironic saying among Chinese developers, which means that by following the "996" work schedule, you are risking yourself getting into the ICU (Intensive Care Unit).
 
 [![Badge](https://img.shields.io/badge/link-996.icu-%23FF4D5B.svg?style=flat-square)](https://996.icu/#/en_US)
+[![Support Ukraine](https://img.shields.io/badge/Support-Ukraine-blue?style=flat&logo=adguard)](https://bank.gov.ua/en/news/all/natsionalniy-bank-vidkriv-spetsrahunok-dlya-zboru-koshtiv-na-potrebi-armiyi)
```

Comment:
As the maintainer of Simple Icons. I must tell you that using [AdGuard](https://simpleicons.org/?q=adgurad)'s trademark as a normal shield icon. Which is not recommended.

Reply:
Remove logo.
Also changes the link to Nonprofits organizations.

Reply:
As an open-sourcerer, I'd rather do something else meaningful than stick "ads" all over the open-source projects.